xml/System.Runtime.InteropServices/DllImportSearchPath.xml

```diff
@@ -137,7 +137,7 @@
       </ReturnValue>
       <MemberValue>2</MemberValue>
       <Docs>
-        <summary>When searching for assembly dependencies, include the directory that contains the assembly itself, and search that directory first. This value is used by the .NET Framework, before the paths are passed to the Win32 <see href="https://go.microsoft.com/fwlink/?LinkId=236091">LoadLibraryEx</see> function.</summary>
+        <summary>When searching for assembly dependencies, include the directory that contains the assembly itself, and search that directory first. When used in NativeAOT, the application's directory is considered the "assembly directory" and searched. This value is used by the .NET Framework, before the paths are passed to the Win32 <see href="https://go.microsoft.com/fwlink/?LinkId=236091">LoadLibraryEx</see> function.</summary>
```

Comment:
Should this talk more generally about single-file deployment models?

Reply:
That is possible. I was looking to @elinor-fung for suggestions along those lines. I'm not sure the best way to express this support. Suggestions are welcome.